### PR TITLE
docs: update contributing docs

### PR DESCRIPTION
**Pull request by alisonjoseph**

### CODE REVIEW

**Review by francinelucca — APPROVED** [review]
👍 👍

File: docs/CONTRIBUTING.md [hunk 1]
```diff
@@ -11,6 +11,11 @@ This project follows a structured format for writing commit messages. The main b
 approach is that we can use these details to automatically generate things like changelogs, in
 addition to clarifying what changes correspond to when looking at our Git history.
 
+Commit message titles will show up in the changelog of each package, so each message should be as
+descriptive as possible.
+
+Feature related write-ups should be outlined in detail within the PR description.
+
```
Comment (jdharvey-ibm) [inline docs/CONTRIBUTING.md:18]:
```suggestion
Commit message titles will show up in the changelog of each package, so each message should be as
descriptive as possible.

Commit message bodies should include all closed and relevant issue numbers. These will show up in
the changelog as well.

Feature related write-ups should be outlined in detail within the PR description.

```

Comment (jdharvey-ibm) [reply docs/CONTRIBUTING.md:18]: (just adding a blurb about issue numbers)

**Comment by alisonjoseph** [comment]
thanks @jdharvey-ibm

**Review by jharvey10 — APPROVED** [review]

**Pull request merged**